src/js/popup/popup.js

```diff
@@ -227,44 +273,70 @@
         sessionState.currentPanel = panelId;
       },
       init: (panelId, data) => {
+        const phonesBtn = document.querySelector(".fx-relay-primary-dashboard-switcher-btn.phone-masks");
+        const masksBtn = document.querySelector(".fx-relay-primary-dashboard-switcher-btn.masks");
+
         switch (panelId) {
           case "custom": 
             popup.panel.masks.custom.init();
+            popup.utilities.buildBackButton("custom", "root", "masks");
             break;
 
-          case "masks": 
+          case "masks":
             popup.panel.masks.init();
+            popup.utilities.setPrimaryPanel(panelId);
+            popup.ariaControls.setSelected(masksBtn);
+            popup.ariaControls.setControls(masksBtn, panelId);
+            break;
+
+          case "phone-masks":
+            popup.panel.phoneMasks.init();
+            popup.utilities.setPrimaryPanel(panelId);
+            popup.ariaControls.setSelected(phonesBtn);
+            popup.ariaControls.setControls(phonesBtn, panelId);
             break;
 
           case "news":
             sendRelayEvent("Panel", "click", "opened-news");
             popup.panel.news.init();
             popup.panel.news.utilities.updateNewsItemCountNotification(true);
+            popup.utilities.buildBackButton("", "root", "masks");
```

Comment:
nit (non-blocking) suggestion: Passing the first argument as `""` is confusing. Is there another option to pass instead of the empty string (`null`, `false`, etc)?